.gitignore

```diff
@@ -2,3 +2,6 @@ out
 node_modules
 format.patch
 Makefile
+
+# production
+build
```

Comment:
Are the generated files put in `builder` folder? or is this for production (I think this is like package file) ?
We have `out` as generated files from `ts` to `js`.
I don't know much about react; is it possible to use `out` ?

Reply:
> Are the generated files put in builder folder? or is this for production (I think this is like package file) ?

the former is correct!

>We have out as generated files from ts to js.
I don't know much about react; is it possible to use out ?

When the react is build, a compressed `output` is produced in the form of html, js, css, etc.
`Build` folder is the folder containing the output.
So, I think there are different kinds.

Reply:
I curious about;
- when making a vscode extension package,  it is OK to have two output folders for artifactoy...

I've found some string is previous draft #93
- src/Visualizer/tsconfig.extension.json
```
        "target": "es6",
        "outDir": "build",
```
and some others
```
      // And restric the webview to only loading content from our extension's `media` directory.
      localResourceRoots: [vscode.Uri.file(path.join(this._extensionPath, 'build'))]
```
```
  private _getHtmlForWebview() {
    const manifest = require(path.join(this._extensionPath, 'build', 'asset-manifest.json'));
    const scriptPathOnDisk = vscode.Uri.file(path.join(this._extensionPath, 'build', mainScript));
    const stylePathOnDisk = vscode.Uri.file(path.join(this._extensionPath, 'build', mainStyle));
```
if `build` is changed to `out`, does it hurt the functionality?

Reply:
> if build is changed to out, does it hurt the functionality?

If you're just changing the name, there's no problem, [this post](https://stackoverflow.com/questions/41495658/use-custom-build-output-folder-when-using-create-react-app#54) mentioned about that
"_Therefore, **I'd recommend you to not use a custom folder naming**, if possible. Try to stick with the default naming._"

Reply:
Thanks for the link!